Gemfile.lock

```diff
@@ -107,6 +107,7 @@ GEM
     backports (3.8.0)
     bcrypt (3.1.11)
     bindata (2.4.0)
+    bluetooth (1.0)
```

Comment:
have you tested what happens when this gets bundled on Windows? We may need a guard here.